cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -319,6 +333,17 @@ public ConfigMap generateMetricsConfigMap() {
         }
     }
 
+    /**
+     * Generates a log ConfigMap according to configured defaults
+     * @return The generated ConfigMap
+     */
+    public ConfigMap generateLogConfigMap(Vertx vertx) {
```

Comment:
So ... I have to admin I kind of hate that we are creating yet another ConfigMap. Can't we use environment variable or rename / reuse the metrics ConfigMap?

Reply:
Yeah, the count of config maps is high. I was worried about the length of env variable, but google says it can be 32,760 chars, so we can use this.

Reply:
In the new commit, (https://github.com/strimzi/strimzi-kafka-operator/pull/512/commits/43882f864520fcf24d553733e3b2dbb645a4aa44) I've merged metrics and logging config maps.